tests/functional/docs/test_shared_example_config.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import json
+
+import botocore.session
+
+
+def test_lint_shared_example_configs():
+    session = botocore.session.Session()
+    loader = session.get_component('data_loader')
+    services = loader.list_available_services('examples-1')
+    for service in services:
+        service_model = session.get_service_model(service)
+        example_config = loader.load_service_model(
+            service, 'examples-1', service_model.api_version
+        )
+        examples = example_config.get("examples", {})
+        for operation, operation_examples in examples.items():
+            for example in operation_examples:
+                yield _lint_single_example, operation, example, service_model
+
+
+def _lint_single_example(operation_name, example_config, service_model):
+    # There should be no non-ascii characters
+    json.dumps(example_config, ensure_ascii=False).encode('ascii')
```

Comment:
It would be nice if we have good error message here (for both conditions you checked).

Things like:

1) What service/operation we failed on.
2) The actual character/value that failed  (I think we do this same check for the .rst example files in the CLI)  You may be able to reuse some of that code.

3) The name of the operation that does not exist.